cutil/initapi/validate.go

```diff
@@ -14,4 +14,22 @@
 
 package initapi
 
-// keep
+import (
+	"fmt"
+	"github.com/kris-nova/kubicorn/apis/cluster"
+)
+
+func serverPoolCounts(initCluster *cluster.Cluster) error {
```

Comment:
Can we break these up into different functions and also give *all* the functions the prefix of `validate`?

For example we want a function called something like `validateServerPoolGreaterThan1` as well as `validateServerPoolMaxCountGreaterThan1` etc etc...

That way we can send another PR that will load these functions dynamically using reflection and we don't have to include the function name in the list anymore!